src/main/kotlin/app/revanced/patches/youtube/layout/branding/header/CustomHeaderPatch.kt

```diff
@@ -48,8 +48,8 @@ object ChangeHeaderPatch : ResourcePatch() {
         ),
         title = "Header",
         description = """
-            The header to use in top bar or the path to a custom header.
-            The path to a folder containing one or more of the following folders matching the DPI of your device:
+            Can be a header name or a path to a custom header folder to use in the top bar.
```

Comment:
"Header name" is kinda difficult to understand

```suggestion
            Either a header name or a path to a custom header folder to use in the top bar.
```

Reply:
> "Header name" is kinda difficult to understand

Perhaps all patch option presets should be referred to as a "preset" or synonymous term

Reply:
This terminology would be introduced here and nowhere else which would be inconsistent. If you want this to change, do so please in a separate PR consistently

Reply:
It's fine for now. Maybe in the future if it's needed.

I don't have a better term than "header name" so I believe this PR is ready to be merged.